### PR TITLE
fix: remove deployment of example app from pipeline

### PR DESCRIPTION
**Pull request by htztomic**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/integration-tests-release.yml [hunk 1]
```diff
@@ -18,16 +18,6 @@ jobs:
           poll-interval-seconds: 10
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  deploy-example-app:
```
Comment (insignias) [inline .github/workflows/integration-tests-release.yml:21]: Comment it instead, may be we want to add it back later ?

File: .github/workflows/merge-develop-to-stage.yml [hunk 2]
```diff
@@ -84,22 +84,10 @@ jobs:
       slack_webhook_url: ${{ secrets.SLACK_WEBHOOK }}
       HOSTED_ZONE_ID: ${{ secrets.HOSTED_ZONE_ID_DEMO }}
       DOMAIN_NAME: ${{ secrets.DOMAIN_NAME_DEMO }}
-
-  deploy-example-app:
```
Comment (maghirardelli) [inline .github/workflows/merge-develop-to-stage.yml:88]: maybe just remove (I agree with Samir to actually comment not delete) in the release workflow. The develop workflow can be refactored in the later ticket to move wbc to rsw. so don't edit this file _yet_

File: .github/workflows/integration-tests-release.yml (hunk 1 above)
Comment (maghirardelli) [reply .github/workflows/integration-tests-release.yml:21]: agree

Comment (maghirardelli) [reply .github/workflows/integration-tests-release.yml:21]: You may actually be able to skip by adding this line to the step:
```
if: false
```

Comment (htztomic) [reply .github/workflows/integration-tests-release.yml:21]: I thought the current ticket is to completely remove example and once wbc is moved into swb, is there any reason to enable example app again?

**Pull request merged**